src/update_engine/postinstall_runner_action.h

```diff
@@ -29,7 +29,7 @@ class ActionTraits<PostinstallRunnerAction> {
 
 class PostinstallRunnerAction : public Action<PostinstallRunnerAction> {
  public:
-  PostinstallRunnerAction() {}
+  PostinstallRunnerAction(std::string app_id);
```

Comment:
With single arg constructors, it's often best to use `explicit` to avoid unexpected type cast from std::string.